functions in JS/main.js

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
move the file to functions directory, and rename the file to functions_in_js_main.html.